examples/libvirt.env

```diff
@@ -1,3 +1,10 @@
+# this is an example configuration file for a libvirt based deployment
+#
+# for a full list of configuration options, see the files in
+# config/*.toml
+
 export ROOKCHECK_HARDWARE_PROVIDER="LIBVIRT"
 export ROOKCHECK_LIBVIRT_CONNECTION="qemu:///system"
 export ROOKCHECK_LIBVIRT_VM_MEMORY="8"
+# a qcow2 image that contains cloud-init. Examples:
+export ROOKCHECK_LIBVIRT_IMAGE = "http://(...)/SLES15-SP2-JeOS.x86_64-15.2-OpenStack-Cloud-GMC.qcow2"
```

Comment:
I'm a bit concerned about this overwriting what is set in caasp.env. It'd be nice for somebody to be able to source both files so they can do a combination of caasp/opensuse and libvirt/openstack etc.

This is why I left it undefined here.

Could also maybe only set this if it is currently unset.

Reply:
It may be better to have both image for caasp (openstack and libvirt) set in the caasp.env indeed

Reply:
That's actually what's happening. So maybe we remove this from here, or at least comment it out with a note.

Reply:
This is very confusing have random env variables, it would be clearer to have single env file per config, like caasp_libvirt.env, caasp_openstack.env

Reply:
Done what Kyr suggested.